public/app/features/dashboard-scene/scene/NavToolbarActions.test.tsx

```diff
@@ -162,6 +162,12 @@ describe('NavToolbarActions', () => {
       const newShareButton = screen.getByTestId(selectors.pages.Dashboard.DashNav.newShareButton.container);
       expect(newShareButton).toBeInTheDocument();
     });
+    it('Should show new export button when newDashboardSharingComponent FF is enabled', async () => {
+      config.featureToggles.newDashboardSharingComponent = true;
+      setup();
+      const newExportButton = screen.getByTestId(selectors.pages.Dashboard.DashNav.NewExportButton.container);
+      expect(newExportButton).toBeInTheDocument();
```

Comment:
great! Could you add in the first test, above, to not expect this button when it's the old design? 
In this test
`Should show old share button when newDashboardSharingComponent FF is disabled`